app.rb

```diff
@@ -0,0 +1,136 @@
+require_relative './person'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # Code to list all books
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224444645-aae6b8f4-9e00-4678-a333-ab50a03e41b0.png)

- [ ] Kindly remove this NOTE to make your readme file descriptive and clean 😊 This NOTE is for you when you create your license file and i can see you already have it so there is not point of having this NOTE in your readme 🤔